src/components/form/ImportModelTemplateForm.vue

```diff
@@ -87,6 +89,35 @@ async function initAllModels() {
     .filter((model) => model.plugin === props.template.plugin)
     .map(({ path }) => path);
 }
+
+/**
+ * Add template config to global config.
+ * @param {string} realModelName - Model name from plugin.
+ * @param {string} content - Template config file content.
+ */
+async function manageConfigFile(realModelName, content) {
+  const originContent = await readProjectFile(
+    props.projectName,
+    { path: 'leto-modelizer.config.json' },
+  ).then((fileInput) => fileInput.content || '');
+
+  let config = {};
+
+  if (originContent !== '') {
+    config = JSON.parse(originContent);
+  }
+
+  config[realModelName] = JSON.parse(content);
```

Comment:
prefere to do:
```js
config[realModelName] = {
   ...config[realModelName],
  ...JSON.parse(content),
};
```

In case, of if we have multiple plugin in same folder.